src/sasmita/EvennumberRange.java

```diff
@@ -0,0 +1,16 @@
+package sasmita;
+
+public class EvennumberRange {
+	
+	void evenNumber() {
+		for(int index=10;index<=15;index++) {
```

Comment:
you need to make this generic, so that same code can be work for different ranges